doc/user/content/sql/system-tables.md

```diff
@@ -0,0 +1,49 @@
+---
+title: "System Tables"
+description: "System tables store metadata about your Materialize instance."
+weight: 14
+menu:
+  main:
+    parent: 'sql'
+---
+
+Like PostgreSQL's `pg_catalog`, Materialize provides a handful of system tables
+containing up-to-date metadata for a given Materialize instance. These tables are
+maintained as [views](../create-view), and can be [queried](../select) as such.
+
+## Details
+
+### mz_databases
+
+`mz_catalog.mz_databases` contains a unique row for each database in a Materialize
+instance.
+
+Field     | Meaning
+----------|----------
+global_id | The unique id of the database.
```

Comment:
nit: when ID is used outside of a variable name, I think "ID" is more appropriate, otherwise looks like the Freudian "id"